src/Symfony/Component/Translation/Bridge/PoEditor/PoEditorProvider.php

```diff
@@ -211,4 +213,52 @@ private function deleteTerms(array $ids): void
             throw new ProviderException(sprintf('Unable to delete translation keys on PoEditor: "%s".', $response->getContent(false)), $response);
         }
     }
+
+    private function ensureAllLocalesAreCreated(TranslatorBagInterface $translatorBag)
+    {
+        $providerLanguages = $this->getLanguages();
+        $missingLanguages = array_reduce($translatorBag->getCatalogues(), function ($carry, $catalogue) use ($providerLanguages) {
+            if (!\in_array($catalogue->getLocale(), $providerLanguages)) {
+                $carry[] = $catalogue->getLocale();
+            }
+
+            return $carry;
+        }, []);
+
+        if ($missingLanguages) {
+            $this->createLanguages($missingLanguages);
+        }
+    }
+
+    private function getLanguages(): array
+    {
+        $response = $this->client->request('POST', 'languages/list');
+
+        if (200 !== $response->getStatusCode() || '200' !== (string) $response->toArray(false)['response']['code']) {
+            throw new ProviderException(sprintf('Unable to list languages on PoEditor: "%s".', $response->getContent(false)), $response);
+        }
+
+        return array_map(function ($language) {
```

Comment:
this can be replaced by `array_column($myArray, 'code')`

Reply:
It has to be done in LokaliseProvider too, I've done it in this PR